src/components/audio_instructions_web/index.js

```diff
@@ -0,0 +1,170 @@
+var Base = require('../base');
+var WebAudioPlayer = require('./web_audio_player');
+
+var AudioInstructionsWeb = Base.extend({
+
+  constructor: function(options) {
+    this.base(options);
+    this.bindAll(this, 'navigationOnRoute', 'onGeopositionUpdate', 'toggleSound');
+
+     //init audio player
+    this.player = new WebAudioPlayer({audioData: options.audioData});
+
+    ffwdme.on('navigation:onroute', this.navigationOnRoute);
+    ffwdme.on('geoposition:update', this.onGeopositionUpdate);
+
+    this.render();
+  },
+
+  lastAction: '',
+
+  player: null,
+
+  iconSoundOff: 'audio_instructions_web/sound-off.svg',
+
+  icon: 'audio_instructions_web/sound-off.svg',
+
+  iconSoundOn: 'audio_instructions_web/sound-on.svg',
+
+  classes: 'ffwdme-components-container ffwdme-grid-w2 ffwdme-grid-h1',
+
+  logoEl: null,
+
+  forwardMeters: 50,
+
+  navigationOnRoute: function(e) {
+
+    var destinationDistance = e.navInfo.distanceToDestination;
+    var distance = e.navInfo.distanceToNextDirection;
+    var nextDirection = e.navInfo.nextDirection;
+
+    if (destinationDistance && destinationDistance < this.forwardMeters && action != this.lastAction){
+      this.player.play('FINISH');
+      this.lastAction = action;
+      return;
+    }
+
+    if (!distance || !nextDirection) return;
+
+    var action = this.audioAction(distance, nextDirection.turnType);
```

Comment:
I think it might make sense to pass the whole direction, not only the street name. This way one could use more information in the TTS than just the street name.

Reply:
done